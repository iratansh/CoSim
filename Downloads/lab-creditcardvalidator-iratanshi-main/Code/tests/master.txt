5199999999999991