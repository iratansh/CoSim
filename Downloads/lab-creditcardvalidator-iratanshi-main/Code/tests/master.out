MasterCard
1199999999999991